extensions/wikia/ArticleVideo/ArticleVideo.i18n.php

```diff
@@ -4,8 +4,10 @@
 
 $messages['en'] = [
 	'articlevideo-watch' => 'watch • $1',
+	'articlevideo-related-video' => 'Related video',
```

Comment:
Why is that? I changed the string for 'watch' also :) I think it's better to set the bullet in css, especially that the duration is out of context of the label and in future this elements may be separated.